docs/ref/models/querysets.txt

```diff
@@ -3400,12 +3400,9 @@ by the aggregate.
         By default, ``StdDev`` returns the population standard deviation. However,
         if ``sample=True``, the return value will be the sample standard deviation.
 
-    .. admonition:: SQLite
+    .. versionchanged:: 2.2
 
-        SQLite doesn't provide ``StdDev`` out of the box. An implementation
-        is available as an extension module for SQLite. Consult the `SQlite
-        documentation`_ for instructions on obtaining and installing this
-        extension.
+        Support was added for SQLite.
```

Comment:
Based on the phrasing used in ref/contrib/gis/geoquerysets.txt... "SQLite support was added."